lib/rules/declaration-block-no-shorthand-property-overrides/__tests__/index.js

```diff
@@ -100,6 +100,10 @@ testRule(rule, {
         "-webKIT-transition",
         "-WEBKIT-transition-property"
       )
+    },
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file.